sdks/java/io/hadoop-input-format/src/test/java/org/apache/beam/sdk/io/hadoop/inputformat/checksum/utils/HIFChecksumEvaluator.java

```diff
@@ -0,0 +1,117 @@
+package org.apache.beam.sdk.io.hadoop.inputformat.checksum.utils;
+
+import java.io.IOException;
+import java.net.InetAddress;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.logging.FileHandler;
+import java.util.logging.Logger;
+import java.util.logging.SimpleFormatter;
+
+import javax.annotation.Nonnull;
+
+import org.elasticsearch.action.search.SearchResponse;
+import org.elasticsearch.client.transport.TransportClient;
+import org.elasticsearch.common.settings.Settings;
+import org.elasticsearch.common.transport.InetSocketTransportAddress;
+import org.elasticsearch.index.query.QueryBuilders;
+import org.elasticsearch.search.SearchHit;
+import org.elasticsearch.transport.client.PreBuiltTransportClient;
+import org.junit.Test;
+
+import com.datastax.driver.core.Cluster;
+import com.datastax.driver.core.ResultSet;
+import com.datastax.driver.core.Row;
+import com.datastax.driver.core.Session;
+import com.google.common.collect.Lists;
+import com.google.common.hash.HashCode;
+import com.google.common.hash.Hashing;
+
+/**
+ * Util class to evaluate checksums.
+ */
+public class HIFChecksumEvaluator {
+  static Logger logger = Logger.getLogger("HIFChecksumEvaluator");
+
+  
+  public void evaluateChecksumForCassandra() throws SecurityException, IOException {
+    List<String> data = new ArrayList<String>();
+    FileHandler fh = new FileHandler("Cassandra.log");
+    logger.addHandler(fh);
+    Cluster cluster = Cluster.builder().addContactPoint("104.154.207.171").build();
```

Comment:
This file is to be removed from the check in. This got checked in with others when we moved to the branch. It was for used by us internally to generate the checksum . i am removing this from the check in.